src/main/java/org/jenkinsci/lib/envinject/service/EnvVarsResolver.java

```diff
@@ -97,45 +98,45 @@ private Node getMasterNode() {
         return computer.getNode();
     }
 
-    public String resolveEnvVars(AbstractBuild build, String value) throws EnvInjectException {
+    public String resolveEnvVars(Run<?, ?> run, String value) throws EnvInjectException {
```

Comment:
🐛  binary compat :-(

I'm going to stop marking the same thing here -- we just need to create methods with the original signature that delegate to this one I think.